apps/src/aichat/views/ChatMessageView.tsx

```diff
@@ -97,19 +97,15 @@ const ChatMessageView: React.FunctionComponent<ChatMessageViewProps> = ({
           const url = getAssetUrl(asset, currentChannelId, levelName);
           return (
             <button
+              key={filename}
```

Comment:
This removes a console warning I was seeing (unrelated to the analytics updates).